msal_extensions/persistence.py

```diff
@@ -199,6 +216,17 @@ def load(self):
                 "DPAPI error likely caused by file content not previously encrypted. "
                 "App developer should migrate by calling save(plaintext) first.")
             raise
+        try:
+            return self._dp_agent.unprotect(data)
+        except OSError as exception:
+            raise PersistenceDecryptionError(
+                err_no=getattr(exception, "winerror", None),  # Exists in Python 3 on Windows
+                message="Decryption failed: {}. "  # pylint: disable=consider-using-f-string
```

Comment:
In our project, we disable all `consider-using-f-string` check as this is really annoying.